packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts

```diff
@@ -29,50 +32,26 @@ enum ReferenceState {
 }
 
 class TestNode {
-    public readonly parents: Set<string> = new Set();
-    public readonly children: Set<string> = new Set();
+    public readonly parents: string[] = [];
+    public readonly children: string[] = [];
     public referenceState: ReferenceState;
     constructor(public readonly id: string, referenceState = ReferenceState.Unreferenced) {
         this.referenceState = referenceState;
     }
-}
-
-class IgnoreErrorLogger extends EventAndErrorTrackingLogger {
-    private readonly ignoredEvents: Map<string, ITelemetryGenericEvent> = new Map();
 
-    public ignoreExpectedEventTypes(... anyIgnoredEvents: ITelemetryGenericEvent[]) {
-        for (const event of anyIgnoredEvents) {
-            this.ignoredEvents.set(event.eventName, event);
-        }
+    public deleteChild(id: string) {
+        const deleteId = this.children.indexOf(id);
```

Comment:
> this.children.indexOf(id); [](http://example.com/codeflow?start=24&length=27)

what if it is not there ?

Reply:
> > * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
> 
> Does it make sense for this file to be shared for all tests ? Not sure but wanted to make sure :)
> 
> Refers to: packages/test/test-gc-sweep-tests/src/testDataObjects.ts:2 in [df0a78b](https://github.com/microsoft/FluidFramework/commit/df0a78b34f160f7f5719e805e21990d63512b583). [](commit_id = [df0a78b](https://github.com/microsoft/FluidFramework/commit/df0a78b34f160f7f5719e805e21990d63512b583), deletion_comment = False)

No, I don't want to publicize this file yet because I don't know if someone would understand the usage of these objects yet. This is very particular to this test. Maybe when it's in a better state we can reuse this. Furthermore, I think it's fine if we keep on rewriting TestDataObjects as it will help other developers understand the framework better.

Reply:
Did some verification to verify that it was there